### PR TITLE
Correct minor grammar error

### PR DESCRIPTION
**Pull request by barbaralaw**
Issue:

In storySort fields table, change 'stories to be show' to 'stories to be shown' for better flow.

### CODE REVIEW

**Review by darleendenno — APPROVED** [review]
Great find ❤️

File: docs/writing-stories/naming-components-and-hierarchy.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ The `storySort` can also accept a configuration object.
 | Field           |  Type   |                       Description                        | Required |      Default Value      |          Example          |
 | --------------- | :-----: | :------------------------------------------------------: | :------: | :---------------------: | :-----------------------: |
 | **method**      | String  | Tells Storybook in which order the stories are displayed |    No    | Storybook configuration |     `'alphabetical'`      |
-| **order**       |  Array  |     The stories to be show, ordered by supplied name     |    No    |    Empty Array `[]`     | `['Intro', 'Components']` |
+| **order**       |  Array  |     The stories to be shown, ordered by supplied name     |    No    |    Empty Array `[]`     | `['Intro', 'Components']` |
```
Comment (darleendenno) [inline docs/writing-stories/naming-components-and-hierarchy.md:106]: We could even update to "The stories to show" to use active voice. Thanks for updating this!

**Comment by jonniebigodes** [comment]
@barbaralaw thanks for the time and effort you've put into the pull request and help us improve our documentation. We appreciate it 🙏 ! And @darleendenno thanks for the excellent suggestion and help with the pull request :)

File: docs/writing-stories/naming-components-and-hierarchy.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/writing-stories/naming-components-and-hierarchy.md:106]: Indeed @darleendenno sounds and reads better that way, appreciate the feedback and improvements.

**Pull request merged**